wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -741,3 +762,23 @@ extension WireCallCenterV3 {
     }
 
 }
+
+extension ZMConversation {
+
+    var avsConversationType: AVSConversationType? {
+        switch (conversationType, messageProtocol) {
```

Comment:
is`messageProtocol` available on this branch?

Reply:
I am not sure, maybe not